concordium-sdk/src/main/java/com/concordium/sdk/responses/bakerlist/BakerId.java

```diff
@@ -0,0 +1,39 @@
+package com.concordium.sdk.responses.bakerlist;
+
+import com.concordium.sdk.responses.AccountIndex;
+import com.concordium.sdk.serializing.JsonMapper;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import concordium.ConcordiumP2PRpc;
+import lombok.Data;
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * Account Index of Baking Account.
+ */
+@Data
+@ToString
+@Getter
+public class BakerId {
+
+    private AccountIndex id;
+
+    @JsonCreator
+    BakerId(long index) {
+        this.id = AccountIndex.from(index);
+    }
+
+    @Override
+    public String toString() {
+        return this.id.toString();
+    }
+
+    public static BakerId[] fromJsonArray(ConcordiumP2PRpc.JsonResponse res) {
```

Comment:
Can you please provide a simple unit test with a mocked JSON response?

Reply:
done